Homeviewserver

Lindayezou opened the pull request.

Commit:
Saves user information in local storage

Refactors the home view to save the facebook id and the user name to local storage upon page load.
gameId still saves to local storage upon game click
 Changes to be committed:
	modified:   ../public/home/home.js

Commit:
Changes gameId to incremented values

GameId to be passed in to the local storage is not a incremented value starting at 1 instead of the _id value generated by mongodb

 Changes to be committed:
	modified:   ../public/home/home.js
	modified:   game/gameController.js

Commit:
Aquires res from post request to make games

Refactors logic to allow new games post request to send proper response to client after refactoring the gameId logic

 Changes to be committed:
	modified:   ../public/home/home.html
	modified:   ../public/home/home.js
	modified:   ../public/services.js
	modified:   game/gameController.js

chououtside commented on server/game/gameController.js:
```diff
@@ -185,37 +185,35 @@ module.exports = {
+          } else {
+            console.error(err);
+          }
+        });
```
newGame.save(function(err) {
    if (err) throw err;
  });

Please get rid of the if (res) we don't need a console log, only to catch the error

Commit:
Refactors error handling for addGame

Removes console.log. Tested before commit.

 Changes to be committed:
	modified:   server/game/gameController.js

chououtside commented:
good to go
